src/openapi/streaming/streaming.ts

```diff
@@ -308,7 +308,7 @@ class Streaming extends MicroEmitter<EmittedEvents> {
      * being rejected so we should get a new one.
      */
     private onUnauthorized() {
-        this.authProvider.tokenRejected();
+        this.authProvider.tokenRejected('streaming');

```

Comment:
Shouldn't streaming URL be provided here so that it's compatible with what `tokenReject` expects? Maybe it's possible to somehow retrieve it from connection. Sims like it could provide some additional context during debugging.

Reply:
ok, it would be nice to correlate between different streaming services...